crime-evidence/src/main/resources/application.yaml

```diff
@@ -67,6 +68,28 @@ spring:
         jwt:
           issuer-uri: ${JWT_ISSUER_URI}
 
+  autoconfigure:
+    exclude: org.springframework.boot.actuate.autoconfigure.tracing.zipkin.ZipkinAutoConfiguration
+
+resilience4j:
+  retry:
+    configs:
+      default:
+        max-attempts: 3
+        wait-duration: 2s
+        enableExponentialBackoff: true
+        exponentialBackoffMultiplier: 2
+        retry-exceptions:
+          - org.springframework.web.reactive.function.client.WebClientRequestException
+          - org.springframework.web.reactive.function.client.WebClientResponseException.BadGateway
+          - org.springframework.web.reactive.function.client.WebClientResponseException.TooManyRequests
+          - org.springframework.web.reactive.function.client.WebClientResponseException.ServiceUnavailable
+          - org.springframework.web.reactive.function.client.WebClientResponseException.GatewayTimeout
+
+    instances:
+      maatCourtDataWebClient:
+        base-config: default
+
 retry-config:
   max-retries: 3
   min-back-off-period: 5
```

Comment:
Since we moved  resilience4j. Can we delete this configuration?

Reply:
Good spot! We shouldnt need this anymore now we've switched do using a different rest client so I've removed the above config.